CTFd/auth.py

```diff
@@ -503,6 +504,18 @@ def oauth_redirect():
 
                 team = Teams.query.filter_by(oauth_id=team_id).first()
                 if team is None:
+                    num_teams_limit = int(get_config("num_teams", default=0))
+                    num_teams = Teams.query.filter_by(
+                        banned=False, hidden=False
+                    ).count()
+                    if 0 < num_teams_limit <= num_teams:
+                        abort(
+                            403,
+                            description="Reached the maximum number of teams ({}). Please join an existing team.".format(
```

Comment:
Can you simplify this with f-strings?

Reply:
I think this was copied and modified from another code section that was using `format()` so I kept it the same for consistency but am happy to change it to f-strings if that's the preferred style for new changes.